packages/utils/src/libp2p/dialHelper.spec.ts

```diff
@@ -49,6 +50,16 @@ async function getNode(id: PeerId, withDHT = false): Promise<Libp2p> {
     }
   })
 
+  const dialProtocol = node.dialProtocol.bind(node)
+
+  node.dialProtocol = async (peer: PeerId | Multiaddr, protocols: string[], options: any) => {
```

Comment:
hacky implementation of relay protocol.

If there is a relay address, call the destination directly using a secret lookup table